index.js

```diff
@@ -162,6 +164,87 @@ const inventoryCommand = (message) => {
   });
 };
 
+const lookUpCommand = (message) => {
+  const originalMessage = message.content;
+  const fuzzySearchKey = originalMessage.slice(8);
```

Comment:
this works -- it is slightly fragile, since changing the `!lookup` command to anything else would result in this function failing. Something safer would be to look at everything after the first " ".

There are many ways to do this, pick one and we can tighten it after